keras_core/utils/image_dataset_utils.py

```diff
@@ -262,8 +253,12 @@ def image_dataset_from_directory(
             num_classes=len(class_names),
             interpolation=interpolation,
             crop_to_aspect_ratio=crop_to_aspect_ratio,
+            data_format=data_format,
         )
 
+        train_dataset = train_dataset.prefetch(tf.data.AUTOTUNE)
```

Comment:
Prefetching should not be moved.

Reply:
Shifted back to after shuffle. But I have a question so why local shuffle is used in `tf.keras` and we are using global shuffle in `keras_core`